src/java/org/apache/cassandra/security/ISslContextFactory.java

```diff
@@ -99,6 +99,15 @@ default boolean hasKeystore()
         return true;
     }
 
+    /**
+     * Returns if this factory uses outbound keystore.
+     *
+     * @return {@code true} by default unless the implementation overrides this
+     */
+    default boolean hasOutboundKeystore() {
```

Comment:
```suggestion
    default boolean hasOutboundKeystore()
    {
```
Minor formatting nit.